File: Prog1/PROG/Lista2/Nova pasta/Ex7.c
#include <stdio.h>
#include <math.h>
#include <locale.h>

#define P 3.14159

int main () {
    setlocale (LC_ALL, "portuguese");
    float area1, perimetro1, area2, perimetro2, areaT, ladoT, ladoQ1, ladoQ2, diagonal;
    int option;
    printf ("\nDigite um numero inteiro qualquer\n");
    scanf ("%d", &option);

    if ((option > 5) || (option <= 0)){
        printf ("\nDigite o valor do lado de um Triangulo Equilatero\n");
        scanf ("%f", &ladoT);
        areaT=(pow(ladoT, 2)*sqrt(3))/4;
        printf ("\nA Area do triangulo é: %.2f", areaT);
        }

    if ((option == 1) || (option == 2)){
        printf ("\nDigite a medida do Lado do Quadrado\n");
        scanf ("%f", &ladoQ1);
        area1=P*(pow((ladoQ1/2), 2));
        perimetro1=(2*P)*(ladoQ1/2);
        printf ("\nA area do Circulo Inscrito é: %.2f\n", area1);
        printf ("\nO Perimetro do Circulo Incrito é: %.2f\n", perimetro1);
        }

    if ((option == 3) || (option == 4) || (option == 5)){
        printf ("\nDigite a medida do Lado do Quadrado\n");
        scanf ("%f", &ladoQ2);
        diagonal=ladoQ2*sqrt(2);
        area2=P*(pow((diagonal/2), 2));
        perimetro2=2*P*(diagonal/2);
        printf ("\nA area do Circulo Circunscrito é: %.2f\n", area2);
        printf ("\nO Perimetro do Circulo Circunscrito é: %.2f\n", perimetro2);
        }

    return 0;

}

File: Prog II/Prog2_teste/Beginner/src/desvio_condicional/Ex_7.java
package desvio_condicional;

import java.util.Scanner;

public class Ex_7 {
	
	public static void main (String[] args) {
		
		Scanner sc = new Scanner(System.in);
		
		final double PI = 3.14159;
		int option;
		double lado, raio_interno, area_1, perimetro_1;
		
		System.out.println("Digite um numero inteiro");
		option = sc.nextInt();

		if (option == 1 || option == 2) {
			System.out.println("Digite o lado do quadrado");
			lado = sc.nextInt();
			raio_interno = lado/2;
			area_1 = PI*(raio_interno*raio_interno);
			perimetro_1 = 2*PI*raio_interno;
			System.out.println(area_1);
			System.out.println(perimetro_1);
		}
		else if (option >= 3 && option <=5) {
			
			System.out.println("Do option 2");
			
		}
		else {
			
			System.out.println("Do option 3");
			
		}
		
	}

}